lib/run.js

```diff
@@ -139,12 +139,15 @@ function run (opts, cb) {
       clearInterval(interval)
       clients.forEach((client) => client.destroy())
       let result = {
+        title: opts.title,
         requests: histAsObj(requests, totalRequests),
         latency: addPercentiles(latencies, histAsObj(latencies)),
         throughput: histAsObj(throughput, totalBytes),
         errors: errors,
         timeouts: timeouts,
         duration: Math.round((Date.now() - startTime) / 1000),
+        start: startTime,
+        finish: Date.now(),
```

Comment:
startTime needs to remain an epoch for the duration calculation, but I use new Date(startTime) to convert back to an object from the epoch. Do you want me to convert startTime to an object and then convert it to an epoch for the duration calculation? it's just two ways of achieving the same thing.